gulpfile.js

```diff
@@ -13,7 +13,9 @@ const gulpConfig = {
   lib: [
     './node_modules/@json-editor/json-editor/dist/**',
     './node_modules/ajv/dist/ajv.min.js',
-    './node_modules/ajv/dist/ajv.min.js.map'
+    './node_modules/ajv/dist/ajv.min.js.map',
+    './node_modules/handlebars/dist/handlebars.js',
```

Comment:
should we only include the minified version of the file?

Reply:
@mariohernandez , good question. We include both minified and non-minified versions of JSON Editor, so I included both versions for Handlebars, to follow that pattern (no pun intended).

Reply:
👍